### PR TITLE
Divyanshu | 85ztdy80v-stage | Sidebar Issue

### PR DESCRIPTION
**Pull request by dvCodeWorld**
https://app.clickup.com/t/85ztdy805

### CODE REVIEW

File: apps/web-giddh/src/app/inventory/components/add-stock-components/inventory.addstock.component.scss [hunk 1]
```diff
@@ -1,3 +1,10 @@
+:host{
+    position: fixed;
+    width: 760px;
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/inventory/components/add-stock-components/inventory.addstock.component.scss:3]: Check without using host

File: apps/web-giddh/src/app/inventory/components/new-branch-transfer/new.branch.transfer.add.component.ts [hunk 2]
```diff
@@ -1333,13 +1337,20 @@ export class NewBranchTransferAddComponent implements OnInit, OnChanges, OnDestr
...
+            },
+            width: '760px',
+            height: '100vh !important'
+        })
         if (!idx) {
             this.getStock();
         }
     }
+    public closeAsideMenuProductServiceModal(): void {
+        this.asideMenuStateForProductService?.close();
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/inventory/components/new-branch-transfer/new.branch.transfer.add.component.ts:1358]: Make documentation for this function

File: apps/web-giddh/src/app/ledger/ledger.component.ts [hunk 3]
```diff
@@ -89,6 +89,8 @@ export class LedgerComponent implements OnInit, OnDestroy {
     @ViewChild('datepickerTemplate', { static: false }) public datepickerTemplate: ElementRef;
     /** Instance of entry confirmation modal */
     @ViewChild('entryConfirmModal', { static: false }) public entryConfirmModal: any;
+     /** Instance of ledger aside pane modal */
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/ledger/ledger.component.ts]: Make format the code

File: apps/web-giddh/src/app/shared/aside-menu-product-service/components/create-stock/sales.create.stock.component.scss [hunk 4]
```diff
@@ -105,3 +105,10 @@ input.focusBorder[type="checkbox"]:focus:after {
     padding: 2px;
     top: 0;
 }
+:host{
+    position: fixed;
+    width: 760px;
+    padding: 0 15px;
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/shared/aside-menu-product-service/components/create-stock/sales.create.stock.component.scss:111]: Remove host or confirm with Ravinder sir

File: apps/web-giddh/src/app/voucher/voucher.component.ts [hunk 5]
```diff
@@ -765,6 +767,7 @@ export class VoucherComponent implements OnInit, OnDestroy, AfterViewInit, OnCha
     private currentTxnRequestObject: Array<any> = [];
     /** Stores the index of current stock variants being loaded */
     private currentlyLoadedStockVariantIndex: number;
+    private
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/voucher/voucher.component.ts]: Remove unused code

File: apps/web-giddh/src/app/voucher/voucher.component.ts [hunk 6]
```diff
@@ -2906,12 +2909,21 @@ export class VoucherComponent implements OnInit, OnDestroy, AfterViewInit, OnCha
...
+        this.asideMenuStateForProductService = this.dialog.open(this.asideMenuProductService, {
+            position: {
+                right : '0',
+            },
+            width: '760px',
+            height: '100vh !important'
+        })
+    }
+
+    public closeAsideMenuProductServiceModal(): void {
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/voucher/voucher.component.ts:2925]: Make documentation for this function

**Review by dilpreetsinghofficial — CHANGES_REQUESTED** [review]

**Review by dilpreetsinghofficial — APPROVED** [review]

**Review by nisha-walkover — APPROVED** [review]

**Pull request merged**